control_toolbox/src/pid_ros.cpp

```diff
@@ -260,6 +260,7 @@ bool PidROS::initialize_from_ros_parameters()
   u_max = UMAX_INFINITY;
   u_min = -UMAX_INFINITY;
   bool antiwindup = false;
+  bool saturation = false;
```

Comment:
```suggestion
  bool saturation = true;
```

Shouldn't the default be true?

https://github.com/ros-controls/control_toolbox/blob/2ad92f1f225918c9c192b8a7e9ebfa3b959ddd10/control_toolbox/src/pid_ros.cpp#L388

TO be coherent with the above one

Reply:
I'd change it in initialize_from_args to be true only if u_min or u_max argument is finite.
for ROS parameters, idk if it should be default false or default true? or use the same logic, depending on i_clamp_max/min parameter?